pkg/apis/operators/v1/catalogsourceconfig_types.go

```diff
@@ -38,6 +38,7 @@ type CatalogSourceConfigList struct {
 
 // CatalogSourceConfigSpec defines the desired state of CatalogSourceConfig
 type CatalogSourceConfigSpec struct {
+	SourceName      string `json:"sourceName"`
```

Comment:
Why SourceName? Why not just `Source`

Also I know that other fields here do not have comments, but I think adding a comment here would be appropriate.

Reply:
I am open to changing the name to `Source` and adding a comment.

Reply:
I am +1 on `Source`. Do we need to consider the `namespace` here for making this future proof?

Reply:
@aravindhp thoughts?

Reply:
> Do we need to consider the namespace here for making this future proof?

Do we want to do this now when `marketplace` doesn't support `OperatorSources` outside of it's namespace?

Reply:
This for when we allow `CatalogSourceConfigs` to also work with `CatalogSources` which could live in other namespaces.

Reply:
Right - but do we want to add this now when we don't support `CSC`s?

Reply:
This is for future proofing to avoid another version bump

Reply:
It seems to me like it would be a second field anyway, rather than one field with a slash in it or something. If that's the case, we could always just add a new field like `SourceNamespace` later and that wouldn't break anything.

Reply:
And this is an open question not a request for change.

Reply:
Yeah I still think we shouldn't necessarily do this now. If we are thinking the source will change then there will be a number of other things that will change and we will most likely not be in the same API landscape anyway. If that's the case then adding a field here may not even make sense anymore.

Reply:
I agree with @kevinrizza on this one.